src/PFire.Infrastructure/Entities/Friend.cs

```diff
@@ -5,10 +5,10 @@ namespace PFire.Infrastructure.Entities
 {
     public class Friend : Entity
     {
-        public int MeId { get; set; }
+        public uint MeId { get; set; }
```

Comment:
I actually don't think EF supports unsigned ints. Let's try to keep all the interacts with the database to continue to use `int`.

There's no realistic concern we'll max out the size anyway

Reply:
Oh definitely, I just covered every base in my changes. I will make a new change in a bit.